ui/apps/platform/src/services/ReportsService.ts

```diff
@@ -99,6 +99,22 @@ export function fetchReportConfigurations(): Promise<ReportConfiguration[]> {
         });
 }
 
+export function fetchReportStatus(id: string): Promise<ReportStatus | null> {
+    return axios
+        .get<{ status: ReportStatus | null }>(`/v2/reports/status/${id}`)
+        .then((response) => {
+            return response.data?.status;
```

Comment:
Yea, I did see it in other places as well and decided to keep it like that. I can see why it was done that way as a guard against a response not having that field. But then again, if we expected `status` and it isn't present, then maybe that indicates a problem with the request itself or the server's response. So this could actually mask that. That might be worth looking into